code/client/makecatalogs

```diff
@@ -268,6 +269,46 @@ def makecatalogs(repopath, options):
                         exit_code = -1
                         continue
 
+            #uninstaller sanity checking
+            do_pkg_check = False
+            uninstaller_type = pkginfo.get('uninstall_method')
+            if uninstaller_type in ['AdobeCCPUninstaller']:
```

Comment:
I think there is at least one other uninstaller_type that uses an uninstaller_item...